.changeset/empty-apples-tie.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-scaffolder': minor
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+Starting the implement the first of the Wizard page for the `next` scaffolder plugin
```

Comment:
"Starting the implementation of the" perhaps? :) There was not much changed in the API report, are people supposed to be able to test this out or is it mostly internal for now?